app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -20,6 +20,8 @@ CONFIG_PM_POLICY_CUSTOM=y
 CONFIG_POWER_DOMAIN=y
 CONFIG_POWER_DOMAIN_INTEL_ADSP=y
 
+CONFIG_ADSP_IMR_CONTEXT_SAVE=y
```

Comment:
@marcinszkudlinski Is there's a way to avoid context save at runtime? Boot with purge-bit set? FYI to @ujfalusi @plbossart

Reply:
What do you mean? IMR context save is a process to keep the system state during D3 - when RAM power is turned off. We don't want to avoid this, but use it